src/Analyzer.Core/Rules/BuiltInRules.json

```diff
@@ -392,10 +392,7 @@
       "allOf": [
         {
           "path": "identity.type",
-          "in": [
-            "UserAssigned",
-            "SystemAssigned"
-          ]
+          "regex": "^(UserAssigned|SystemAssigned|SystemAssigned, *UserAssigned)$"
```

Comment:
The [documentation for this property](https://learn.microsoft.com/en-us/azure/templates/microsoft.web/sites?pivots=deployment-language-arm-template#managedserviceidentity-1) uses exactly one space, and doesn't specify any other details, so I would actually err on the side of caution and only allow for exactly one space, as the doc states. You can then change this operator back to `in` for better readability.

Reply:
Sounds good, that will align with the Bicep build warning as well when the space is missing.